### PR TITLE
[CONS BLOG] Will Python Be #1 Forever?

### PR DESCRIPTION
**Pull request by bskinn**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [ ] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by ericdatakelly** [comment]
The plot doesn't match the text in the second paragraph.  He refers to Cobol, for example, but the plot does not contain Cobol.  Should we fix this or retain the original text?  One option is to add a second plot (adding too many languages to the first plot makes it too busy).

**Comment by bskinn** [comment]
@ericdatakelly This effort is *just* for migrating the posts in their existing state. Please note in the comment thread here things that need changes, as you've done, and we'll create new content issues out of these PRs for downstream discussion and possible action.

File: apps/consulting/posts/will-python-be-1-forever.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+can maintain the premier position in language popularity indefinitely. I’m sure
+there were many who believed that Java would never be replaced as #1.
+
+![](/posts/will-python-be-1-forever/python-1-img-1.jpg)
+
+_([Image source: PYPL][pypl page])_
+
+But that certainly does not mean that Python is going away completely. Virtually
+none of the languages that have come to prominence in the past have disappeared
+yet. They continue on as legacy code. Looking at the bottom right of the same
```
Comment (ericdatakelly) [inline apps/consulting/posts/will-python-be-1-forever.md:29]: The text leads the reader through the plot, but the languages described in the text are not present in the plot.  Consider replacing the plot, adding a plot, or modifying the text.

Comment (ericdatakelly) [reply apps/consulting/posts/will-python-be-1-forever.md]: That looks a lot better.  Thanks!

**Pull request merged**